x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SLMSnapshotBlockingIntegTests.java

```diff
@@ -340,14 +340,18 @@ private void testUnsuccessfulSnapshotRetention(boolean partialSuccess) throws Ex
             logger.info("--> waiting for snapshot to complete");
             successfulSnapshotName.set(snapshotResponse.get().getSnapshotName());
             assertNotNull(successfulSnapshotName.get());
-            Thread.sleep(TimeValue.timeValueSeconds(10).millis());
```

Comment:
Since this runs twice, this saves almost 20s for this test suit overall :)

Reply:
Thanks, I think this was something I added for debugging originally and didn't remove properly.